src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -229,6 +231,11 @@ public function load(array $configs, ContainerBuilder $container)
             $this->registerRequestConfiguration($config['request'], $container, $loader);
         }
 
+        if (null === $config['csrf_protection']['enabled']) {
```

Comment:
Will this ever be reached? I would have expected the `treatNullLike()` call above to result in the value here being `true`.

Reply:
verified: yes! "defaults" applies after "treatNullLike" (thus, this applies only to explicit user values)

Reply:
maybe we should add a test for this :)

Reply:
that's a test for the Config component :)

Reply:
I meant for different configs here, but it will become quite hard as the `FullStack` class is always available.

Reply:
`treatNullLike` is on the `csrf_protection` node, not on the `enabled` node.